tests/test_client.py

```diff
@@ -1,15 +1,45 @@
 import unittest
 from gepetto_viewer_rerun import Client
+from gepetto_viewer_rerun.entity import Group
```

Comment:
If you don't want it to be included in that module, you must not add it into the module: everything is public in python.
If you need it only in tests, you can define it in a file in the `tests/` directory, outside of `src`.

Otherwise, I think it is not a big issue if we have it in the module.

Reply:
Even if i define a class `Group` with same attributes, methods, ... in my test file, it won't be same Group as in `gepetto_viewer_rerun` module right ?